appengine/flexible/pubsub/app.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+runtime: ruby
+env: flex
+entrypoint: bundle exec ruby app.rb
+instance_class: F2
+runtime_config:
+  operating_system: ubuntu22
+  runtime_version: 3.2
```

Comment:
We talked about this early today (for a .NET discussion), with James Ma and Matthew. I think our decision can be generalized to all the 3 silver languages. We believe it's better to fully rely on Ruby's idiomatic ways to specify the runtime version. Basically, making Flex Ruby, .NET and PHP runtimes versionless.

Google's topic search results showed me this:
`
You can customize the Ruby version for your app by adding a . ruby-version file or by setting the ruby directive in your Gemfile . If a version is set in both places . ruby-version will take priority.
`

Do you know if our Ruby buildpacks support these two ways?

Reply:
Both these options work for specifying the ruby version.